script.js

```diff
@@ -1,12 +1,105 @@
 //You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+let arrayOfEpisodes = getAllEpisodes();
+let mainEl = document.querySelector("main");
+let searchBar = document.querySelector("#searchInput");
+let displayNumberOfEpisodes = document.querySelector("#displayEpisodesText");
+let selectEl = document.querySelector("#selectMenu");
+
+//search bar
+searchBar.addEventListener("keyup", (e) => {
+  const searchString = e.target.value.toLowerCase();
+  const filteredEpisodes = arrayOfEpisodes.filter((episode) => {
+    return (
+      episode.summary.toLowerCase().includes(searchString) ||
+      episodeCode(episode).toLowerCase().includes(searchString) ||
```

Comment:
Check for summary and name only. No need to check for episodeCode(episode)
As stated:
2.
Add a "live" search input:
i. Only episodes whose summary OR name contains the search term should be displayed